ChallengeTestRunner/challenge_test_lib/challenge_test.py

```diff
@@ -262,9 +262,14 @@ def challenge_fun(self, *args, **kwargs):
         # We need the __class__ because in that way it doesn't pass self as argument to the function
         return self.__class__.CHALLENGE_FUN(*args, **kwargs)
 
-    def assert_printed(self, value):
+    def assert_printed(self, value, index=None):
         str_value = str(value)
-        return str_value in self.mock_print.printed
+        if index is not None:
+            return str_value in self.mock_print.printed[index]
+
+        filtered = [printed for printed in self.mock_print.printed
+                    if str_value in printed]
+        return len(filtered) > 0
```

Comment:
To be consistent with the other asserts, this method should raise an AssertionError if it does not pass in the test.
I think a good solution would be to replace:

`return len(filtered) > 0`

by 

`self.assertTrue(len(filtered) > 0)`

This would use the default error message, so it would be more elegant to also have a `msg` parameter such as [this](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertFalse).